Ai Booking Hotel Assistant

syed-saad-zahidi opened the pull request:
[Hey Edd . I tried to make a booking assistant with tools .also i used booking.com api from rapid api. please add this in community contribution

ed-donner commented:
Hey - I'm super grateful that you're worked on this. It looks like this might not be ready to be merged - please can I ask you to review this:
https://edwarddonner.com/pr
I very much look forward to merging when this is addressed. Thanks so much
Ed

syed-saad-zahidi commented:
Hey Ed , I have moved my booking assistant in week 2 community contribution folder. Also removed unnecessory commits. Please accept my PR now , Thanks
